.github/workflows/release.yml

```diff
@@ -6,123 +6,25 @@ on:
-          uses: docker/login-action@v3
-          with:
-            username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
-            password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
-        
-        - name: Build and push docker image
-          uses: docker/build-push-action@v6
-          with:
-            platforms: ${{ env.DOCKER_PLATFORMS }}
-            context: .
-            push: true
-            tags: |
-              ${{ env.DOCKER_IMAGE_NAME }}:${{ env.VERSION }}${{ env.TAG_SUFFIX }}
-        
-        - name: Push latest tag
-          if: ${{ ! github.event.release.prerelease }}
-          uses: docker/build-push-action@v5
-          with:
-            platforms: ${{ env.DOCKER_PLATFORMS }}
-            context: .
-            push: true
-            tags: ${{ env.DOCKER_IMAGE_NAME }}:latest
-        
-        - name: Update title for successful pre-release
-          if: ${{ github.event.release.prerelease }}
-          env:
-            GH_TOKEN: "${{ secrets.COREINT_BOT_TOKEN }}"
-          run: |
-            gh release edit ${{ env.TAG }} --title "${{ env.TAG }}"
-
-  notify-failure:
-    if: ${{ always() && failure() }}
-    needs: [release-integration]
-    runs-on: ubuntu-latest
-    steps:
-      - name: Notify failure via Slack
-        uses: archive/github-actions-slack@master
-        with:
-          slack-bot-user-oauth-access-token: ${{ secrets.COREINT_SLACK_TOKEN }}
-          slack-channel: ${{ secrets.COREINT_SLACK_CHANNEL }}
-          slack-text: "❌ `${{ env.ORIGINAL_REPO_NAME }}`: [image release failed](${{ github.server_url }}/${{ env.ORIGINAL_REPO_NAME }}/actions/runs/${{ github.run_id }})."
-
-  update-title-on-failure:
-    if: ${{ always() && failure() }}
-    needs: [release-integration]
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
+  container-release:
+    uses: newrelic/coreint-automation/.github/workflows/reusable_image_release.yaml@v3
+    with:
+      original_repo_name: "newrelic/nri-statsd"
+      docker_platforms: "linux/amd64,linux/arm64"
+      docker_image_name: "newrelic/nri-statsd"
+      go_version_file: "tests/integration/go.mod"
       
-      - if: ${{ github.event.release.prerelease }}
-        name: Reflect failure in pre-release title
-        env:
-          GH_TOKEN: "${{ secrets.COREINT_BOT_TOKEN }}"
-        run: |
-          gh release edit ${{ github.event.release.tag_name }} --title "${{ github.event.release.tag_name }} (pre-release-failure)"
+      run_integration_tests: true
+      run_nix_unit_tests: false
+      run_windows_unit_tests: false
       
-      - if: ${{ ! github.event.release.prerelease }}
-        name: Reflect failure in release title
-        env:
-          GH_TOKEN: "${{ secrets.COREINT_BOT_TOKEN }}"
-        run: |
-          gh release edit ${{ github.event.release.tag_name }} --title "${{ github.event.release.tag_name }} (release-failure)"
\ No newline at end of file
+      use_build_push_action: true
```

Comment:
This seems to be used only here, I would change it to remove such option from the action